sceptre/template_handlers/file.py

```diff
@@ -24,12 +24,8 @@ def schema(self):
         }
 
     def handle(self):
-        project_path = self.stack_group_config.get("project_path")
         input_path = Path(self.arguments["path"])
-        if input_path.is_absolute():
-            path = str(input_path)
-        else:
-            path = str(Path(project_path) / 'templates' / input_path)
```

Comment:
@alexharv074, you don't want to cut this out. The whole point is that we WANT to support absolute paths... for paths that are absolute; But _relative_ paths should resolve to the sceptre_dir/templates directory.

Reply:
@jfalkenstein My observation was that in both cases, `input_path` will already contain the resolved path by this stage, but let me play some more to see if I can find a counter example.

Reply:
Ok, I did more testing. See below. Refactored, so closing this discussion.